design/use_cases/indoor_localization_use_case.md

```diff
@@ -0,0 +1,15 @@
+# Indoor Localization
+As a Robot user I want my robot to know its location on a given map of an indoor area so that it can move around the area
```

Comment:
Robot location, is that metric (3.12, 7.42), topologic (node 5), semantic maybe (near the fridge) ... ?

Reply:
@artivis - should the user specify or care about what metric to use for robot location?

Reply:
The user may not care about specifying explicitly what metric to use but he/she may care about to it when requesting feedback about the robot location and/or when sending it a navigation goal.

Reply:
So, sounds like there are a couple use case-level interactions here between the Robot User and the Robot: Determine Current Robot Location and Send Robot to Location. Then, in the context of these interactions, there might be details about how the location is to be specified.